src/machine/machine_rp2040_uart.go

```diff
@@ -0,0 +1,132 @@
+// +build rp2040
+
+package machine
+
+import (
+	"device/rp"
+	"runtime/interrupt"
+)
+
+// UART on the RP2040.
+type UART struct {
+	Buffer    *RingBuffer
+	Bus       *rp.UART0_Type
+	Interrupt interrupt.Interrupt
+}
+
+func (uart *UART) Configure(config UARTConfig) error {
```

Comment:
Can you add some documentation to the public functions `Configure` and `SetBaudRate`?

Reply:
Done.